report_thesis/src/sections/background/linear_and_regularization_models/lasso.tex

```diff
@@ -1,7 +1,11 @@
 \subsubsection{Least Absolute Shrinkage and Selection Operator (LASSO) Regression}
-Having introduced ridge regression, we now turn our attention to the \gls{lasso} regression based on \citet{James2023AnIS}.
+A disadvantage of ridge regression is that it includes all features in the final model.
+While it does shrink the coefficients towards zero, it does not eliminate any.
+This may not impact prediction accuracy but makes interpretation harder, especially with many variables.
+For example, in datasets with numerous features, we might prefer a model that includes only the most important ones.
+However, ridge regression includes all features, and while increasing the penalty reduces coefficient sizes, it does not exclude variables\cite{James2023AnIS}.
```

Comment:
When taking this line + 3 into account, it does actually sound like the features get removed. So maybe clarify?
'Effectively excluded, as their coefficients are set to 0' or something.

Reply:
```suggestion
However, ridge regression includes all features, and while increasing the penalty reduces coefficient sizes, it does not set any coefficients to zero\cite{James2023AnIS}.
```